src/config/dynamic-theme-fixes.config

```diff
@@ -2813,6 +2813,15 @@ INVERT
 
 ================================
 
+codingame.com/ide/*
+
+CSS
+.cg-statement .statement-body span.var, .cg-statement .statement-body var {
+    background-color: rgb(0, 10, 164);
```

Comment:
```suggestion
   color: var(--darkreader-neutral-background) !important;
```

Reply:
I thought that changing the text colour would remove from the logic of the page, since the original text color doesn't change. But that works too.